ddtrace/contrib/redis/util.py

```diff
@@ -90,7 +92,6 @@ def _trace_redis_execute_pipeline(pin, config_integration, resource, instance):
     ) as span:
         span.set_tag(SPAN_MEASURED_KEY)
         span.set_tag(redisx.RAWCMD, resource)
-        span.set_tags(_extract_conn_tags(instance.connection_pool.connection_kwargs))
```

Comment:
I guess we don't actually need to make this change now since we don't use it in aioredis <2.0 (and can revert it in aredis/redis).